queries.sql

```diff
@@ -8,3 +8,39 @@ select name, escape_attempts from animals where weight_kg>10.5
 select * from animals where neutered is true
 select * from animals where name != 'Gabumon'
 select * from animals where weight_kg between 10.4  and 17.3
+
+
+UPDATE animals 
+SET species = 'digimon'
+WHERE name like '%mon%' 
+
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species is null
+
+
+-- transactions
+begin;
+delete from animals
+
+rollback
```

Comment:
- [ ] @JakariaJishan kindly add query before rollback to verify that change was madeand after the rollback to verify that change was undone.